src/hooks/useMergedRefs.ts

```diff
@@ -26,7 +26,7 @@ export function assignRef<RefValueType = any>(
   ref: AssignableRef<RefValueType> | null | undefined,
   value: any,
 ) {
-  if (ref === null) return;
+  if (ref === null || ref === undefined) return;
```

Comment:
```suggestion
  if (ref == null) return;
```

Shortcut that TS is (probably) aware of